src/components/FeatureWalkthrough/video.ts

```diff
@@ -0,0 +1,9 @@
+export interface IVideo {
+    mp4: string
+    webm: string
+}
+
+export interface IVideoElement {
+    el: HTMLVideoElement | null
+    paused: boolean
+}
```

Comment:
```suggestion
```

Reply:
This file was deleted while removing the collocated typescript files: b8af56f